### DISCUSSION TITLE
Power Meter Einstellungen fuer SDM (Modbus)

### DISCUSSION THREAD

**Discussion by Manos1966** [hoylabs/OpenDTU-OnBattery#488] [Q&A]
Hat jemand ein paar Infos wie die SDM Verbindung funktioniert? 
Braucht man eine separate Platine (TTL RS-485) und falls Ja, welche (und gibt ein Pin-Out dafuer)?

![RS485 Modbus](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/3faa5ac8-3e72-49be-9642-d81fa2d43e5d)

**Comment by spcqike** [comment]
ja du brauchst einen Wandler

https://github.com/helgeerbe/OpenDTU-OnBattery/pull/140

Und, wenn ich es richtig lese, sind die Pins fest auf 13/32 gelegt, oder? 
https://github.com/helgeerbe/OpenDTU-OnBattery/blob/4ee49a6ecb90a800e57db7554a88b53cb0bf55af/include/PowerMeter.h#L12C1-L18

**Reply by Manos1966** [reply]
Vielen Dank @spcqike 
(es ist nur fuer die Wiki) 

Ich habe etwas gefunden
[How to interface ESP32 with RS485 (Modbus)](https://embeddedthere.com/how-to-interface-esp32-with-rs485-modbus-sensors-with-example-code/)

Es sieht so aus, als ob ich auch drei Pins brauche.

![esp8266-nodemcu-rs458-max3485-module-connection_bb](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/f2146bb7-d4fe-4102-a796-4dc5ce9ef1b6)
